.github/workflows/pr-build-live-branch.yml

```diff
@@ -0,0 +1,125 @@
+name: Build live branch for PR
```

Comment:
I can change this but I just want it consistent with the same concept made by Jetpack and Woo core. So that it can be easier to recognize across A8C: 

- Jetpack Live Branch PCYsg-goI-p2
- WooCommerce Live Branch pcShBQ-BZ-p2

Reply:
I chose a balanced approach here: update the GitHub workflow name 3214566f40c904aa013f0358a9a7bcd876359c72, but keep the filename to have its consistency with [Woo core](https://github.com/woocommerce/woocommerce/blob/31ec8d8352a5a2b904eb299a32997ebfa961236f/.github/workflows/pr-build-live-branch.yml).